src/services/createCommunity.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+const getTokenFromLocalStorage = () => {
+  const storedData = localStorage.getItem('persist:auth');
+  if (!storedData) return null;
+
+  const parsedData = JSON.parse(storedData);
+  const accessToken = parsedData.accessToken;
+
+  return accessToken ? accessToken.replace(/"/g, '') : null;
+};
+
+export const createCommunity = async (userData) => {
+  const token = getTokenFromLocalStorage();
+  if (!token) {
+    throw new Error('No token found in local storage');
+  }
+
+  try {
+    const response = await axios.post(`${process.env.REACT_APP_API_BASE_URL}/api/v1/communities/`, userData, {
```

Comment:
Create variable for API_BASE_URL on top of component, before the functions, so we can use it with other functions also.

Reply:
The best approach is to use the reusable function, AuthenticatedAPIClient, which creates the axion object with the attached authentication token. Here is the example: https://github.com/salman299/react-communi-verse/blob/develop/src/views/communities/index.js#L24